internal/pkg/cli/interfaces.go

```diff
@@ -644,6 +644,11 @@ type workloadTemplateGenerator interface {
 		*clideploy.GenerateCloudFormationTemplateOutput, error)
 }
 
+
+type Runner interface {
```

Comment:
Seems like we can keep this interface private:
```suggestion
type runner interface {
```